src/protocols/bitcoin/components/TransferSendForm.vue

```diff
@@ -269,6 +272,19 @@ export default defineComponent({
       { deep: true },
     );
 
+    watch(
+      activeAccount,
```

Comment:
Is it possible to change the account when the transfer send modal is open? If not I think it makes no sense to watch activeAccount here.